client/state/initial-state.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import debugModule from 'debug';
+import localforage from 'localforage';
+
+/**
+ * Internal dependencies
+ */
+import { createReduxStore } from 'state';
+import { reducer } from 'state';
+import { TO_OBJECT, FROM_OBJECT } from 'state/action-types'
+
+/**
+ * Module variables
+ */
+const debug = debugModule( 'calypso:state' );
+
+const localforageConfig = {
+	name: 'calypso',
+	storeName: 'calypso_store',
+	description: 'Calypso Storage'
+};
+
+function toObject( state ) {
+	return reducer( state, { type: TO_OBJECT } );
+}
+
+function fromObject( state ) {
+	return reducer( state, { type: FROM_OBJECT } );
+}
+
+function loadInitialState( initialState ) {
+	debug( 'loading initial state with', initialState );
+	return createReduxStore( fromObject( initialState ) );
```

Comment:
I'm not sure it would give us much. I'd prefer not to change this, unless people feel very strongly that I try and avoid calling the reducer directly.

On flip side, we wouldn't want to dispatch a `SERIALIZE` event as it would modify the store, triggering another call to `persistOnStore`, and if we'd managed to escape the on change callbacks we'd still need to trigger a `DESERIALIZE` action to get us back into the right state.